Code/Framework/AzToolsFramework/AzToolsFramework/ToolsComponents/EditorEntityIconComponent.cpp

```diff
@@ -210,32 +222,48 @@ namespace AzToolsFramework
 
```

Comment:
maybe not nescessary, but I would add an assert to check that the path is not empty before returning

Reply:
It is allowed to be empty, in which case a caller can decide what to do with it.  The renderer for example uses a default icon.